src/editor/profiler_ui.cpp

```diff
@@ -378,8 +378,8 @@ struct ProfilerUIImpl : public ProfilerUI
 		Lumix::Array<float> m_frames;
 		struct Hit
 		{
-			Lumix::uint64 start;
```

Comment:
Yeah, I do not like that bgfx is "leaking" in engine, but it would require a lot of boilerplate code to abstract it. By the way IntervalGraph is LumixEngine's custom code created and used only for this purpose, so it can be changed if necessary. 

> (otherwise there is bunch of error messages from usages of bgfx which uses cstdint's ints.

Was is something about unsigned long to unsigned long long conversion? 

I do not like that [there is a little type mismatch now.](https://github.com/nem0/LumixEngine/blob/master/src/editor/profiler_ui.cpp#L490)